### ISSUE TITLE
[SUGGESTION] New features for the fluid amulets

### ISSUE THREAD

**Issue by Nova001Sol** [sinkillerj/ProjectE#472]
Alright, so I tend to make the Evertide Amulet fairly early on, usually before I even bother building the collectors and such. The Volcanite Amulet follows shortly after. I use both of them to help me mine in their respective fluids, and to sneak up on unsuspecting friends*. I have three ideas for features that would make them far more useful though.
-Improved sight: as I mentioned earlier, the Evertide and Volcanite Amulets (mostly the latter) are important tools when I'm mining. They help me search for materials in their fluids, and that includes stuff the DM pickaxe doesn't insta-mine with right-click such as dense ores and beehives and such. But their use there is mostly limited to keeping me... alive. While that's useful and all, I would really like to see better in lava and water without needing a constant supply of night vision potions. Also, I've just discovered night vision does NOT help in lava, it only makes everything look brighter orange.
-Improved mobility while submerged: faster swimming, that is all. It would make the Amulets great tools both for exploring the great colorful depths and keeping the arrows from my friends* out of my avatar's butt. Horizontal speed boosts would be neat, faster rising and sinking (perhaps only while holding shift?) even more so. An absolute must? No, but it definitely saves time while swimming around and the resources required to make a Rending Gale.
-Having them act like a bucket: removes the need of figuring out which button created a projectile again. Right-click to place water or lava where you're aiming. I don't know why, but having to use my keyboard to place a block of water just feels... wrong somehow.
Thank you for reading.

**Comment by goreae** [comment]
I agree with all of these. The amulets used to act like a bucket in EE2, but they don't in PE for whatever reason. And I haven't found any item in any mod that allows you to see clearly in/through lava. I mean you could use a sponge or something, but that sort of defeats the purpose of having the amulet.

**Comment by MaPePeR** [comment]
They also might be lacking the ability to fill a fluid container
